tests/api_test.py

```diff
@@ -13,14 +13,16 @@
 # limitations under the License.
 import base64
 import unittest
+import six.moves.urllib.parse as urllib_parse
+
 import requests
 
 from flexmock import flexmock, flexmock_teardown
 from hamcrest import assert_that, equal_to, raises, calling
 import requests_mock
 from requests_mock.exceptions import NoMockAddress
-from ubersmith_client import api
```

Comment:
Make sure that this style of import still works even if you proved it with the exceptions.